scripts/createEDM4hepFile.py

```diff
@@ -373,6 +377,124 @@
+    frame = podio.Frame()
+
+    frame.put(create_EventHeaderCollection(VECTORSIZE), "EventHeader")
+
+    particles = frame.put(create_MCParticleCollection(), "MCParticleCollection")
+    particle = particles[0]
+
+    hits = frame.put(
+        create_SimTrackerHitCollection(particle), "SimTrackerHitCollection"
+    )
+    simtracker_hit = hits[0]
+
+    calo_contribs = frame.put(
+        create_CaloHitContributionCollection(particle), "CaloHitContributionCollection"
+    )
+    calo_contrib = calo_contribs[0]
+
+    hits = frame.put(
+        create_SimCalorimeterHitCollection(calo_contrib), "SimCalorimeterHitCollection"
+    )
+    simcalo_hit = hits[0]
+
+    frame.put(create_RawCalorimeterHitCollection(), "RawCalorimeterHitCollection")
+
+    hits = frame.put(create_CalorimeterHitCollection(), "CalorimeterHitCollection")
+    calo_hit = hits[0]
+
+    clusters = frame.put(
+        create_ClusterCollection(VECTORSIZE, calo_hit), "ClusterCollection"
+    )
+    cluster = clusters[0]
+
+    hits = frame.put(create_TrackerHit3DCollection(), "TrackerHit3DCollection")
+    tracker_hit = hits[0]
+    frame.put(create_TrackerHitPlaneCollection(), "TrackerHitPlaneCollection")
+
+    frame.put(create_RawTimeSeriesCollection(VECTORSIZE), "RawTimeSeriesCollection")
+
+    tracks = frame.put(
+        create_TrackCollection(VECTORSIZE, tracker_hit), "TrackCollection"
+    )
+    track = tracks[0]
+
+    pids, pid = create_ParticleIDCollection(VECTORSIZE)
 
-    writer.write_frame(frame, "events")
+    vertices, vertex = create_VertexCollection(VECTORSIZE)
+
+    reco_particles = frame.put(
+        create_ReconstructedParticleCollection(vertex, cluster, track),
+        "ReconstructedParticleCollection",
+    )
+    reco_particle = reco_particles[0]
+
+    vertex.addToParticles(reco_particle)
+    frame.put(vertices, "VertexCollection")
+
+    pid.setParticle(reco_particle)
+    frame.put(pids, "ParticleIDCollection")
+
+    put_link_collection(frame, "RecoMCParticleLink", reco_particle, particle)
+    put_link_collection(frame, "CaloHitSimCaloHitLink", calo_hit, simcalo_hit)
+    put_link_collection(
+        frame, "TrackerHitSimTrackerHitLink", tracker_hit, simtracker_hit
+    )
+    put_link_collection(frame, "CaloHitMCParticleLink", calo_hit, particle)
+    put_link_collection(frame, "ClusterMCParticleLink", cluster, particle)
+    put_link_collection(frame, "TrackMCParticleLink", track, particle)
+    put_link_collection(frame, "VertexRecoParticleLink", vertex, reco_particle)
+
+    frame.put(create_TimeSeriesCollection(VECTORSIZE), "TimeSeriesCollection")
+    frame.put(create_RecDqdxCollection(track), "RecDqdxCollection")
+
+    frame.put(
+        create_GeneratorEventParametersCollection(VECTORSIZE, particle),
+        "GeneratorEventParametersCollection",
+    )
+    frame.put(create_GeneratorPdfInfoCollection(), "GeneratorPdfInfoCollection")
+
+    return frame
+
+
+if __name__ == "__main__":
+    import argparse
+    import sys
```

Comment:
Imports at the top, don't the python tools complain about this?

Reply:
Not configured for this repo